src/track/cueinfoimporter.h

```diff
@@ -35,7 +32,7 @@ class CueInfoImporter {
             const audio::SignalInfo& signalInfo);
 
   private:
-    QList<CueInfo> m_cueInfos;
+    const QList<CueInfo> m_cueInfos;
```

Comment:
this will not only inhibit moving but also Qt's reference counting optimizations.

Reply:
The object is not moved. What reference counting optimizations are you referring to? The reference counter is always mutable even in a const object.

Reply:
> The object is not moved.

The CueInfoImporter automatically gets a move-constructor generated by the compiler which will recursively move the member variables if possible. Making the member variable const will make it immovable, forcing a copy even when the object is moved. 

> What reference counting optimizations are you referring to?

Qt's implicit sharing and copy-on-write optimizations for containers.

> The reference counter is always mutable even in a const object.

True, though my former point still applies.

Reply:
Thank you.